core-models/src/main/scala/com/mesosphere/usi/core/models/PodRecord.scala

```diff
@@ -0,0 +1,27 @@
+package com.mesosphere.usi.core.models
+
+import java.time.Instant
+
+/**
+  * Pod record is an immutable event of what happened to all pods.
+  * @param sequenceNr number used for ordering the pod events.
```

Comment:
Not sure if I understand. Is this a sequential id?

Reply:
I don't understand this either